stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -128,23 +125,17 @@ def __init__(
         self.gamma = gamma
         self.train_freq = train_freq
         self.gradient_steps = gradient_steps
-        self.n_episodes_rollout = n_episodes_rollout
         self.action_noise = action_noise
         self.optimize_memory_usage = optimize_memory_usage
 
         # Remove terminations (dones) that are due to time limit
         # see https://github.com/hill-a/stable-baselines/issues/863
         self.remove_time_limit_termination = remove_time_limit_termination
 
-        if train_freq > 0 and n_episodes_rollout > 0:
-            warnings.warn(
-                "You passed a positive value for `train_freq` and `n_episodes_rollout`."
-                "Please make sure this is intended. "
-                "The agent will collect data by stepping in the environment "
-                "until both conditions are true: "
-                "`number of steps in the env` >= `train_freq` and "
-                "`number of episodes` > `n_episodes_rollout`"
-            )
+        if isinstance(self.train_freq, int):
+            self.train_freq = (self.train_freq, "step")
```

Comment:
what do you think of using NamedTuple? so later you can do `train_freq.n` and `train_freq.type` instead of `train_freq[1]` ?

Reply:
Sure but can I have `train_freq.quantity` and `train_freq.unit`?

Reply:
`quantity` sounds a bit weird to me... I think you wrote `frequency` in the docstring, no?

Reply:
You are right. I was borrowing terminology from [here](https://en.wikipedia.org/wiki/Unit_of_measurement) and messed it up. In our case the `train_freq` is the quantity, your `n` is the magnitude unit of measurement must be either `step` or `episode` (but this could be extended in the future). So I propose `train_freq.magnitude` and `train_freq.unit`.

Reply:
I would really for go `train_freq.frequency` or `train_freq.interval` (so discrete values, that's why I proposed the `n` at first, from the maths side) as `magnitude` sounds like a continuous value to me.
and yes for the `train_freq.unit` ;)